src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/ModuleEnvironment.cpp

```diff
@@ -44,14 +44,17 @@ void SetApplicationEnvironmentVariables(_In_ IHttpServer &server, _In_ IHttpCont
 
     IHttpServer2* server2;
     if (SUCCEEDED(HttpGetExtendedInterface(&server, &server, &server2))) {
+        // GetAppPoolConfigFile likely returns a 0 terminating string but the SAL annotations don't state that, we should probably change the code
+        // just to be safe
+#pragma warning(suppress: 6387)
```

Comment:
Maybe I should remove this suppression so that when we re-enable the rule it'll flag this case and we won't forget to fix it